Client/Frontend/Browser/TopTabsLayout.swift

```diff
@@ -80,6 +80,12 @@ class TopTabsViewLayout: UICollectionViewFlowLayout {
         }
     }
 
+    override func layoutAttributesForSupplementaryView(ofKind elementKind: String, at indexPath: IndexPath) -> UICollectionViewLayoutAttributes? {
+        let attributes = super.layoutAttributesForSupplementaryView(ofKind: elementKind, at: indexPath)
+        attributes?.zIndex = -2
```

Comment:
A constant would be good here (I see we do this in a few places) -even though I have written collection views a few times, zIndex seems odd since one doesn't typically need to use this. Then we can comment on it in a single location.